scripts/clear_completed_tasks.sh

```diff
@@ -1,7 +1,7 @@
-#! /bin/bash
+#!/bin/bash
```

Comment:
I'm not saying you should change it, but if this was going to be changed anyway, you should use the more portable shebang `#!/usr/bin/env bash`.